### ISSUE TITLE
RANGE_DELETE does not accept more than one DataIDs to be removed

### ISSUE THREAD

**Issue by fvanagthoven** [facebook/relay#994]
The docs @ https://facebook.github.io/relay/docs/guides-mutations.html 
indicate that you can remove multiple nodes from a connection, without removing the records from the store:

> Given a parent, a connection, one or more DataIDs in the response payload, and a path between the parent and the connection, Relay will remove the node(s) from the connection but leave the associated record(s) in the store.

` deletedIDFieldName: string | Array<string>`
>The field name in the response that contains the DataID of the removed node, or the path to the node removed from the connection

Error: 

```
app.js:30019 Uncaught Invariant Violation: writeRelayUpdatePayload(): Expected `removedHabitIDs` to be a string, got `["SGFiaXQ6NTZmNTVmYWNiNmQwMjgwZDYxN2IwYjI2"]`.
```

NODE_DELETE does work as specified, but I'd like to use RANGE_DELETE. :)

I might be wrong. But I think I'm doing it right.

```js
getConfigs() {
  return [{
    type: 'RANGE_DELETE',
    parentName: 'sprint',
    parentID: this.props.sprint.id,
    connectionName: 'activeHabitPath',
    deletedIDFieldName: 'removedHabitIDs',
  }];
}
```

**Comment by NevilleS** [comment]
The docs are definitely misleading here. `NODE_DELETE` has a fairly simple logic like "if the result is an array, loop through and delete each record ID", something similar for `RANGE_DELETE` would be good.

**Comment by sampepose** [comment]
Fixed with 766a459529418e3816adda98cb866e55b6ac27fd
